OracleWebCenterSites/dockerfiles/12.2.1.4/Dockerfile

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2019, 2020 Oracle and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# Description: This file is used to build Oracle WebCenter Sites image
+#
+# ORACLE DOCKERFILES PROJECT
+# -------------------------------------------------------------
+# This is the Dockerfile for Oracle WebCenter Sites 12.2.1.4 Generic Distro
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# -------------------------------------------------------------
+# (1) V983400-01.zip
+#     Download the Oracle WebCenter Sites 12c R2 (12.2.1.4.0) installer from https://www.oracle.com/middleware/technologies/webcenter-sites/downloads.html
+#
+# HOW TO BUILD THIS IMAGE
+# -------------------------------------------------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -f Dockerfile -t oracle/wcsites:12.2.1.4 . 
+
+# Pull base image from the Oracle Container Registry or Docker Store and tag the name as 'oracle/fmw-infrastructure:12.2.1.4'
+# -------------------------------------------------------------
+FROM oracle/fmw-infrastructure:12.2.1.4 as builder
+
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+USER root
+ENV SITES_CONTAINER_SCRIPTS=/u01/oracle/sites-container-scripts \
+	SITES_INSTALLER_PKG=wcs-wls-docker-install \
+	SITES_PKG=V983400-01.zip \
+	SITES_JAR=fmw_12.2.1.4.0_wcsites.jar
+	
+# Copy packages and scripts 
+# -------------------------------------------------------------
+COPY sites-container-scripts/* $SITES_CONTAINER_SCRIPTS/
+COPY $SITES_INSTALLER_PKG /u01/$SITES_INSTALLER_PKG
+COPY $SITES_PKG install.file oraInst.loc /u01/
+
+#Install packages and adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# -------------------------------------------------------------
+
+RUN yum install -y hostname && \
```

Comment:
Do you need the `hostname` package to install?

Reply:
yes we need hostname package for reading the ip address in scripts.